manifest.php

```diff
@@ -46,14 +46,16 @@
     ),
     'install' => array(
         'php' => [
-            SetupProctoringEventListeners::class
+            SetupProctoringEventListeners::class,
+            \oat\taoAct\scripts\install\RegisterAuthProvider::class,
```

Comment:
it's \oat\ltiProctoring\scripts\install\RegisterAuthProvider, and you could put it in the use part